lib/modules/showImages.js

```diff
@@ -298,8 +307,21 @@ addModule('showImages', (module, moduleID) => {
 		const maximumTop = viewportHeight * (bufferScreens + 1);
 		const minimumBottom = viewportHeight * bufferScreens * -1;
 
-		imageList.forEach(image => {
-			if (image.imageLink && image.imageLink.media) {
+		activeImageList.forEach(image => {
+			// If image has no expandoBox, remove from this list
+			if (!image.expandoBox) {
+				activeImageList.splice(activeImageList.indexOf(image), 1);
```

Comment:
Ah, still not fully got my head around all the new ES6 stuff :p Set's api makes a lot more sense with what i'm doing :+1: